src/lib.rs

```diff
@@ -379,5 +382,6 @@ pub fn new_vm() -> RootedThread {
     ::vm::channel::load(&vm).expect("Loaded channel library");
     ::vm::debug::load(&vm).expect("Loaded debug library");
     ::io::load(&vm).expect("Loaded IO library");
+    ::regex_bind::load(&vm).expect("Loaded regex library");
```

Comment:
If you want to make regex optional you will also need to write two wrapper load functions. Then that wrapper is callable regardless if the `regex` feature is active or not.

```rust
#[cfg(features ="regex")]
fn load_regex(vm: &Thread) {
 ::regex_bind::load(&vm).expect("Loaded regex library")
}
#[cfg(not(features ="regex"))]
fn load_regex(_: &Thread) {
}
```